themes/phenomic-theme-base/content/index.md

```diff
@@ -1,32 +1,43 @@
 ---
 title: Phenomic base theme
 layout: Homepage
+hero: https://farm4.staticflickr.com/3949/15589950511_3675b15e59_k.jpg
+# hero credit: https://www.flickr.com/photos/pslee999/15589950511/
+cta:
+  label: About this theme
+  link: "about"
 ---
 
-Hi there.
+Hi there. Before having fun with Phenomic, be sure to read the documentation :
 
-Before having fun with Phenomic, be sure to read (or at least pretend to)
-the documentation :
-
-* [Setup](https://phenomic.io/docs/setup/)
-* [Getting Started](https://phenomic.io/docs/getting-started/)
-* [Usage](https://phenomic.io/docs/usage/)
+* [How to setup a new project with Phenomic](https://phenomic.io/docs/setup/)
+* [Getting Started with Phenomic](https://phenomic.io/docs/getting-started/)
+* [Usage & APIs](https://phenomic.io/docs/usage/)
 * [FAQ](https://phenomic.io/docs/faq/)
 
 Also take a look to
 [existing websites that use Phenomic](https://phenomic.io/showcase/),
 some projects share their source code!
 
-Built with
-<a href="https://facebook.github.io/react/">
-  <img alt="" src="assets/react.svg" width="16" height="16" />
-  React
-</a>.
+<!-- demo to show you that you can use "assets" folder -->
+<small>
+  Built with
+  <a href="https://facebook.github.io/react/">
+    <img alt="" src="assets/react.svg" width="16" height="16" />
+    React
+  </a>.
+</small>
 
 ---
 
-You may want to take a look to the <a href="/404.html">404.html</a>.
+You may want to take a look to the [404.html](/404.html) and tweak the render.
+
+If you want to adjust page layouts, directly adjust them in ``src/layouts/``.  
+If you want to adjust small parts like header, footer, buttons, etc, you will
+find those in ``src/components/``.  
+For colors and common CSS variables, you will find stuff to adjust in ``postcss.plugins.js``.
 
```

Comment:
I think there's too many adjusts here, it doesn't read all that well. Here's an alternate version:

> If you need to make adjustments, you can directly alter page layouts in ``src/layouts/``; small things like the header, footer, buttons and so on can be changed in ``src/components/``; and for colors and common CSS variables, you can modify ``postcss.plugins.js``.